documentation/_puppetdb_nav.html

```diff
@@ -39,6 +39,7 @@ <h2 id="puppetdb">PuppetDB {{ puppetdb_version }}</h2>
       <li><a href="/puppetdb/{{ puppetdb_version }}/scaling_recommendations.html">Scaling Recommendations</a></li>
       <li><a href="/puppetdb/{{ puppetdb_version }}/repl.html">Debugging with Remote REPL</a></li>
       <li><a href="/puppetdb/{{ puppetdb_version }}/load_testing_tool.html">Load Testing</a></li>
+      <li><a href="/puppetdb/{{ puppetdb_version }}/resource_statuses.html">Resource Statuses **PE FEATURE**</a></li>
```

Comment:
docs stance is to use Puppet Enterprise on first use in a document. After that PE is ok.